Assignment2/BonzoM-fromJSONtoJSON-LD.jsonld

```diff
@@ -0,0 +1,18 @@
+{
+  "@context": {
+    "ejemplo":"http://example.org/",
+    "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+    "vcard": "http://www.w3.org/2006/vcard/ns#",
+    "name": "rdfs:label",
+    "hometown": "rdfs:label",
+    "age": "rdfs:label",
+    "almamater": "rdfs:label"

```

Comment:
Syntactically is ok, but you have to define each attribute with a specific vocabulary attribute name because when you want to retrieve, for example your age, if you query "rdfs:label" you will not be able to distinguish what data you ask for. If you have your own vocabulary you can use it to define your attributes e.g. "name":"yourvocab:name", "age":"yourvocab:age", etc.